src/insights_client/tests/test_motd.py

```diff
@@ -0,0 +1,113 @@
+import os
+import pathlib
+import tempfile
+import typing
+import unittest
+import unittest.mock
+
+import pytest
+
+import insights_client
+
+
+class MockFS:
+    """Mock filesystem.
+
+    Ensures that the files required for the MOTD manipulation are created
+    in a temporary directory instead of working with actual system files.
+    """
+
+    _dir: tempfile.TemporaryDirectory
+
+    @property
+    def _chroot(self) -> pathlib.Path:
+        return pathlib.Path(self._dir.name)
+
+    def touch(self, path: str):
```

Comment:
Although the [touch](https://linux.die.net/man/1/touch) command is often (mis)used to create a file, it’s primary intended purpose is to set a timestamp. Since this method only mimics the side-effect and not the original function, the name is misleading.

Reply:
from `touch (5)`:

> Update the access and modification times of each FILE to the current time.
> A FILE argument that does not exist is created empty,

The first paragraph documents this behavior. It is not misleading, `touch` is really common when you want to create an empty file on UNIX-like systems.